.devcontainer/post-create.sh

```diff
@@ -1,51 +1,21 @@
 #!/bin/sh
 
-# Add welcome message
-sudo cp .devcontainer/welcome.txt /usr/local/etc/vscode-dev-containers/first-run-notice.txt
+# this runs at Codespace creation - not part of pre-build
 
-# Switch to vets-api ruby version
-export PATH="${HOME}/.asdf/shims:${HOME}/.asdf/bin:${PATH}"
-asdf install ruby $( cat .ruby-version )
-asdf global ruby $( cat .ruby-version )
+echo "post-create start"
+echo "$(date)    post-create start" >> "$HOME/status"
 
-git clone https://github.com/department-of-veterans-affairs/vets-api-mockdata.git ../vets-api-mockdata
+# update the repos
+git -C /workspaces/vets-api-mockdata pull
+git -C /workspaces/vets-api pull
 
-sudo apt update
-sudo apt install -y libpq-dev pdftk shared-mime-info postgresql-15-postgis-3
+{
+{
+  "rubyLsp.rubyVersionManager": "none"
+}
+} >> /home/vscode/.vscode/settings.json
```

Comment:
This isn't working as expected. Might need to put it in the repo dir instead and gitignore it.